OrbitVulkanLayerClientGgp/orbit_capture_vulkan_layer_config.pb.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# proto-file: layer_config.proto
+# proto-message: orbit_vulkan_capture_protos.LayerConfig
+
+capture_service_arguments {
+  functions: []
+  file_directory: ""
+  log_directory: ""
+  sampling_rate: 0
```

Comment:
This is thought to be a template for user provided config files, right?

I'd put the defaults from OrbitVulkanLayerClientGgp/LayerData.cpp into this file so when someone wants to use this there is a reasonable value they can alter.

Reply:
Yes, this is just the skeleton as example. I decided not to put the default values mainly because some of the default values are not set in the layer (e.g sampling_rate and log_directory) and frame_time_threshold_ms would not be as exact as the one set in the layer 16.66 !=  1000.0 / 60.0. Should I just go with this approximation and leave the non default ones as they are?

Reply:
Alternatively I could add a comment on the right showing the default values as a reference.

Reply:
I think the accuracy is not an issue at all. Especially since it is related to timing and that is a bit handwavy anyways.

Even if you put in just some arbitray working values e.g. for sampling_rate I think this is helpful to get a user started.

Reply:
I have included example values (corresponding to the default ones) in the config file :)